conanfile.py

```diff
@@ -0,0 +1,26 @@
+from conans import ConanFile, CMake, tools
+import ptvsd
+
+class NanorangeConan(ConanFile):
+    name = "nanorange"
+    version = "0.1.0"
+    license = "BSL-1.0"
+    author = "Tristan Brindle (tcbrindle at gmail dot com)"
+    url = "https://github.com/tcbrindle/NanoRange"
+    description = "Range-based goodness for C++17"
+    exports_sources = ("CMakeLists.txt", "single_include/*", "cmake/nanorangeConfig.cmake.in", "LICENSE_1_0.txt")
+    generators = "cmake"
+
+    def package(self):
+        ptvsd.break_into_debugger()
```

Comment:
Do you intend to check in this line to break into the debugger?  Look like this might be here while you were debugging the packaging step of the conan configuration.